.github/workflows/chromatic-build.yml

```diff
@@ -95,29 +95,16 @@ jobs:
         # Generate snapshots for only changed stories
         # See: https://www.chromatic.com/docs/turbosnap
         onlyChanged: true
-        debug: true
 
-    # (if) Run this step on dependabot PRs.
-    - name: Skip Chromatic builds (dependabot PRs only)
```

Comment:
**note:** Decided to drop the `release` case and instead merge these two "targets" into a single `bot` one.